components/connector-service/docs/api/connectorapi.yaml

```diff
@@ -128,7 +128,7 @@ components:
       properties:
         url:
           type: 'string'
-          example: 'https://cert-service.kyma.cluster.cx/v1/remoteenvironments/ec-default/info?token=1edfc34g'
```

Comment:
The file `docs/application-connector/docs/assets/connectorapi.yaml` should be the same as this one.